test/modules/test_xml/01_Green_to_pink_shipment.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`epcis-otjson-transpiler.test.js` extracts data from importers file, you can use that logic if you need files for unit tests